copyright

```diff
@@ -1264,6 +1264,12 @@ Copyright: Becca Tommaso
 License: CC-BY-SA-4.0
 Comment: Derived from works by Nate Graham (under the same license).
 
+Files:
+ images/outfit/heliarch?license*
+Copyright: Becca Tommaso
+License: CC-BY-SA-4.0
+Comment: Derived from works by Michael Zahniser (under the same license).
```

Comment:
Is this really derived from works by Michael Zanhiser? It doesn't look similar to anything in the game, maybe its just auto included a bit too easily.

Reply:
Its .xcf file has the same texture as other Coalition outfits and ships (it's called Feuerverzinkte Oberfläche in the xcf), so I figured I'd have that in to be safe. If that's not needed just because of using a texture MZ used too, what should the comment be?

Reply:
No comment, maybe? I asked beccabunny he said the "derived from MZ" was not needed.